05/app.js

```diff
@@ -1,3 +1,54 @@
-console.log('DOM');
-
 const curr = document.querySelector('.js-curr');
+
+const btnSibling = document.createElement('button');
+btnSibling.textContent = 'Usuń z koszyka';
+
+// 1.
+curr.parentNode.insertBefore(btnSibling, curr.nextSibling);
+
+// 2.
+function newClassForSiblings(element, parent) {
+	const children = [...parent.children];
+	const filteredSiblings = children.filter((child) => child !== element);
+	filteredSiblings.map((el) => el.classList.add('siblings'));
```

Comment:
Należałoby wykluczyć element `curr`.

PS. Jeśli nie tworzyć nowej tablicy to zamiast `.map()` lepiej wykorzystać `.forEach`